pkgs/development/libraries/cyrus-sasl-xoauth2/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, libtool, automake, cyrus_sasl }:
+
+with lib;
```

Comment:
I would put this on the `meta` attrset, since that's the only place this is actually used.

Reply:
Good point. Fixed.